apps/publisher/themes/default/theme.js

```diff
@@ -537,9 +543,9 @@ var engine = caramel.engine('handlebars', (function() {
                 var type = options.hash.type;
                 var tenantId = options.hash.tenantId;
                 var username = options.hash.username;
-                var isAuthorized =options.hash.auth ? options.hash.auth : false; 
+                var isAuthorized =options.hash.auth ? options.hash.auth : false;

```

Comment:
Thanks @splinter , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/d132865cb7ce4b43f78012710e79d2772dab7797)